diff --git a/feeds_excel/libraries/phpExcelReader/Excel/reader.php b/feeds_excel/libraries/phpExcelReader/Excel/reader.php
index 90d21a5..df17232 100644
--- a/feeds_excel/libraries/phpExcelReader/Excel/reader.php
+++ b/feeds_excel/libraries/phpExcelReader/Excel/reader.php
@@ -28,7 +28,7 @@
 
 
 //require_once 'PEAR.php';
-require_once 'Spreadsheet/Excel/Reader/OLERead.php';
+//require_once 'Spreadsheet/Excel/Reader/OLERead.php';
 //require_once 'OLE.php';
 
 define('SPREADSHEET_EXCEL_READER_BIFF8',             0x600);
@@ -967,7 +967,7 @@ class Spreadsheet_Excel_Reader
     {
         if ($numValue > 1) {
             $utcDays = $numValue - ($this->nineteenFour ? SPREADSHEET_EXCEL_READER_UTCOFFSETDAYS1904 : SPREADSHEET_EXCEL_READER_UTCOFFSETDAYS);
-            $utcValue = round(($utcDays+1) * SPREADSHEET_EXCEL_READER_MSINADAY);
+            $utcValue = round(($utcDays) * SPREADSHEET_EXCEL_READER_MSINADAY);
             $string = date ($this->curformat, $utcValue);
             $raw = $utcValue;
         } else {